packages/rancher-kiali-server/rancher-kiali-server.patch

```diff
@@ -136,32 +135,30 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-kiali-server/charts-original/v
 +web_root_override: true
 +
  #
- # Settings that mimic the Kiali CR which are placed in the ConfigMap
- #
-@@ -57,10 +60,10 @@
+ # Settings that mimic the Kiali CR which are placed in the ConfigMap.
+ # Note that only those values used by the Helm Chart will be here.
+@@ -34,10 +37,10 @@
    custom_dashboards:
      excludes: ['']
      includes: ['*']
 -  image_name: quay.io/kiali/kiali
 +  repository: rancher/kiali-kiali
    image_pull_policy: "Always"
    image_pull_secrets: []
--  image_version: v1.22.0
-+  tag: v1.22.0
+-  image_version: v1.23.0
++  tag: v1.23.0
    ingress_enabled: true
    node_selector: {}
    override_ingress_yaml:
-@@ -182,3 +185,13 @@
-   web_fqdn: ""
+@@ -66,3 +69,11 @@
+   metrics_enabled: true
+   metrics_port: 9090
```

Comment:
Whats this part ?

Reply:
its comes from the upstream chart - they mad significant changes to the values.yaml

Reply:
But this is in the patch, are we making changes to the new values.yaml ? Otherwise it would just get imported automatically. Is it the web_fqdn ?

Reply:
https://github.com/kiali/helm-charts/commit/ac6d43c03f30225210181650a83573a49d62ea3a, yes, the web_fqdn was removed.

Reply:
nvm, this is fine.